descriptors/ModuleDescriptor-template.json

```diff
@@ -328,7 +328,10 @@
           "permissionsRequired": ["inventory.holdings.item.put"],
           "modulePermissions": [
             "inventory-storage.holdings.item.get",
-            "inventory-storage.holdings.item.put"
+            "inventory-storage.holdings.item.put",
+            "inventory-storage.holdings.collection.get",
+            "inventory-storage.holdings.batch.post",
+            "user-tenants.collection.get"
```

Comment:
"user-tenants.collection.get" is used by mod-orders-storage Item Update Kafka consumer in ECS mode:
https://github.com/folio-org/mod-orders-storage/pull/442/files#diff-f34125a4cad28f81be682f398500f5bb7697e4763bb10edbe8bc3a6d83423ca5R41